src/sync/arc.rs

```diff
@@ -25,8 +25,30 @@ impl<T> Arc<T> {
     }
 
     /// Returns the inner value, if the `Arc` has exactly one strong reference.
-    pub fn try_unwrap(_this: Arc<T>) -> Result<T, Arc<T>> {
-        unimplemented!();
+    pub fn try_unwrap(this: Arc<T>) -> Result<T, Arc<T>> {
```

Comment:
this probably needs a
```suggestion
    #[track_caller]
    pub fn try_unwrap(this: Arc<T>) -> Result<T, Arc<T>> {
```
on it so that the `location!()` passed to `get_mut` has the callsite in the user code where `try_unwrap` was called.